packages/siop-oid4vp/lib/__tests__/AuthenticationRequest.request.spec.ts

```diff
@@ -511,30 +605,33 @@ describe('create Request JWT should', () => {
   it('succeed when requesting with a valid PD', async () => {
     const opts: CreateAuthorizationRequestOpts = {
       version: SupportedVersion.SIOPv2_ID1,
-      /*payload: {
+      payload: {
```

Comment:
Why is the payload fully re-enabled, whilst this test already contains a request object. If you need to add things add them there

Reply:
reverted.
I guess I must have been testing with a broken pex or something

Reply:
actually test "should throw error if presentation definition object is not valid" fails without the payload
It expects 
> Presentation definition in the request claims is not valid

but receives

> Received message:   "referenceUri specified, but object to host there is not present"

      14 |     if (opts.reference_uri) {
      15 |       // reference URI, but no actual payload to host there!
    > 16 |       throw Error(SIOPErrors.REFERENCE_URI_NO_PAYLOAD)
         |             ^
      17 |     } else if (checkRequestObject) {
      18 |       throw Error(SIOPErrors.BAD_PARAMS)
      19 |     }


There is a payload in the opts root, but that doesn't end up in the requestObject